### DISCUSSION TITLE
Дaunching the application and constantly giving an error.Рow can I fix this error ?

### DISCUSSION THREAD

**Discussion by odessa341** [lllyasviel/Fooocus#2916] [Q&A]
![Снимок экрана 2024-05-10 224442](https://github.com/lllyasviel/Fooocus/assets/169399752/349b85e6-281b-40dc-a8d7-d7db07f18c7d)

**Comment by 15xa** [comment]
i have the same problem

**Reply by mashb1t** [reply]
@odessa341 please check the terminal output for errors.
